.github/workflows/build-and-push-to-docker.yml

```diff
@@ -15,16 +38,14 @@ jobs:
         - docker/py/Dockerfile
         - docker/r/Dockerfile
         - docker/cuda/Dockerfile
-        - docker/cuda-tf/Dockerfile
-        - docker/generic/Dockerfile
```

Comment:
did you mean to remove this one?

Reply:
Yes - `generic` is not referenced in the `Makefile` ; it is referenced in the github action but just for linting afaict - it does not look like an image that is actually built.